### ISSUE TITLE
[bug] word form composition helper not functioning properly

### ISSUE THREAD

**Issue by rcochrane55** [DraqueT/PolyGlot#1192]
The word form composition helper doesn't seem to work for irregular verbs. For example, in my Germanic conlang, if I select a separable-prefix verb in the word form composition helper it will show me wordforms for that word as if it were a regular weak verb and the prefix wasn't separable, rather than the handwritten irregular forms with separated prefix. It uses only the auto generated wordforms and completely ignores any overrides in the conjugations.

**Comment by DraqueT** [comment]
Sorry for the delay in replying here, but is this still a problem for you? If so, I can take a look and see what I can do to resolve this for you. :)

**Comment by rcochrane55** [comment]
Yep, I have not updated my system or install and it is still a problem. It should be fairly easy to reproduce. Set up an automatically generated conjugation, then override the autogenerated wordforms. then check the word form composition helper and see that the autogenerated forms are used rather than the overrides.
